website/src/pages/dictionary/index.js

```diff
@@ -64,6 +63,7 @@ export const ModalPortal = ({ children }) => {
 };
 
 const data = require('./data.json');
+const treeData = require('./tree.json');
```

Comment:
just a question, maybe not scope of this PR, but is this meant to change with selected schema?

Reply:
@hlminh2000 I will address this in this PR. nice catch thanks! :)

Reply:
Adding tree data to state to mirror how we load current dictionary data.
We can look at moving this data generation to runtime if need be